packages/dapp/src/components/Navbar/NavbarManagement.tsx

```diff
@@ -2,55 +2,58 @@ import MenuIcon from '@mui/icons-material/Menu';
 import { Typography } from '@mui/material';
 import { WalletManagementButtons } from '@transferto/shared/src';
 import { ChainSwitch } from '@transferto/shared/src/atoms/ChainSwitch';
-import { ThemeSwitch } from '@transferto/shared/src/atoms/ThemeSwitch';
-import { useSettings } from '@transferto/shared/src/hooks';
 import { useEffect, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
-import { SubMenuKeys } from '../../const';
+import { shallow } from 'zustand/shallow';
 import { useChainInfos } from '../../providers/ChainInfosProvider';
-import { useMenu } from '../../providers/MenuProvider';
 import { useWallet } from '../../providers/WalletProvider';
+import { useMenuStore, useSettingsStore } from '../../stores';
+import { ThemeSwitch } from '../ThemeSwitch';
 import {
   NavbarDropdownButton,
   NavbarManagement as NavbarManagementContainer,
 } from './Navbar.style';
 
 const NavbarManagement = () => {
-  const anchorRef = useRef<HTMLButtonElement>(null);
-  const settings = useSettings();
-  const menu = useMenu();
+  const mainMenuAnchor = useRef<any>(null);
```

Comment:
I'm sure there is a type for this 🙂

Reply:
Well yes! But looking at I also found that this is not necessary and can be deleted. Anyways,
All those "any" in various places have to be types out correctly. This is another task @